libraries/lib-numeric-formats/NumericConverterType.h

```diff
@@ -10,9 +10,10 @@
  **********************************************************************/
 #pragma once
 
-enum class NumericConverterType
-{
-   TIME,
-   FREQUENCY,
-   BANDWIDTH,
-};
+#include "Identifier.h"
+
+using NumericConverterType = Identifier;
```

Comment:
This change is good, and to make it even better, used `TaggedIdentifier`.

`using NumericConverterType = TaggedIdentifier<struct NumericConverterTypeTag>`

(The tag is an "elaborated type" that you don't need to define)

This makes a type distinction between this and all other `TaggedIdentifier` specializations so they won't impliciatly interconvert.

Note, Identifier.h contains several `TaggedIdentifier` specializations, but do NOT add to that list.  I don't like that list and need to move those aliases elsewhere after fixing some still remaining legacy confusions about the right levels for certain declarations.  (See also lib-components, a library with an unclear purpose.)